api/clients/v2/payload_retriever.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/Layr-Labs/eigenda/api/clients/codecs"
 	"github.com/Layr-Labs/eigenda/api/clients/v2/verification"
 	"github.com/Layr-Labs/eigenda/common/geth"
-	contractEigenDABlobVerifier "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDABlobVerifier"
+	verifierContract "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDABlobVerifier"
```

Comment:
```suggestion
	verifiercontract "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDABlobVerifier"
```

Reply:
as ugly as it is convention better to use all caps, otherwise `verifierContract` looks like its an actual struct, not a module name. Maybe use `verifier_contract` if you prefer.

Reply:
Your comment says "all caps", but suggestion shows all lowercase.

I went with all lowercase for now [f7008c51](https://github.com/Layr-Labs/eigenda/pull/1157/commits/f7008c5128ae10ebd46f069ccbeb2bf01125b1b8)

Reply:
did mean lowercase yes oops